scripts/check-config.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+
+var fs = require('fs');
+var path = require('path');
+var chalk = require('chalk');
+
+var configPath = path.resolve('../config.json');
+
```

Comment:
Do we really need to bring in this additional NPM 'chalk' for colorizing console output?  We would like to try to keep the number of packages we bring in to a minimum, since the consumers of the DocuSign-Node-Client package also have to bring in these packages too.  Not seeing chalk add a whole lot of value here.